test/functional-portable/datastoresrp/cloud/resources/redis_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/radius-project/radius/test/rp"
+	"github.com/radius-project/radius/test/step"
+	"github.com/radius-project/radius/test/testutil"
+	"github.com/radius-project/radius/test/validation"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Redis_Recipe(t *testing.T) {
```

Comment:
I'm missing something. why is this moved to cloud resources tests?

Reply:
It is using an Azure provider in the environment. Please see: https://github.com/radius-project/radius/blob/b961aa02f67375eb5e809559cfad77f8d5509c6f/test/functional-portable/datastoresrp/cloud/resources/testdata/datastoresrp-resources-redis-recipe.bicep.

Reply:
Why though? Does it actually require an Azure resource? 

We should have a justification when tests require the cloud.